cli/azd/internal/repository/infra_confirm.go

```diff
@@ -103,10 +102,16 @@ func (i *Initializer) infraSpecFromDetect(
 		if svc.Docker == nil || svc.Docker.Path == "" {
 			// default builder always specifies port 80
 			serviceSpec.Port = 80
-
 			if svc.Language == appdetect.Java {
 				serviceSpec.Port = 8080
 			}
+		} else {
+			ports := svc.Docker.ExposedPorts
```

Comment:
Just leaving this as a note, when you get to this requirement:

If more than one port is exposed, prompt for a selection (console.Select) with the multiple ports available, and an "Other" selection to provide the port number.

Feel free to ping me directly on Teams if you have any questions here.

Reply:
Updated.